packages/create-gatsby/src/components/utils.ts

```diff
@@ -16,6 +16,8 @@ export function center(
   padding = ` `,
   width = DEFAULT_WIDTH
 ): string {
-  const pad = padding.repeat(Math.round((width - stringLength(text)) / 2))
+  const count = Math.round((width - stringLength(text)) / 2)
```

Comment:
❓ I'm not sure if I should take abs value or fallback to 0 when count is less than 0?